tests/unit/engine/actor_test.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+from unittest.mock import Mock
+from unittest.mock import patch
+
+from engine import actor
+from engine.globals import G
+from engine import room
+from engine import tartarus
+from tests import common
+
+
+class ActorTest(common.EngineTest):
+
+    @patch('engine.actor.say.insayne')
+    def test_player_dies(self, mock_say):
+        abattoir = room.Room.create()
+        abattoir.add_character(G.player)
+
+        with self.assertRaises(tartarus.RaptureException):
+            G.player.die(cause='misadventure')
+            mock_say.assert_called_once_with('You die.')
+            mock_say.assert_called_once_with('...')
+        self.assertEqual('misadventure', G.cause_of_death)
+
+    def test_npc_dies(self):
+        lou_dobbs = actor.create_actor(10, 10, 10, 10, 10, 10, 10, 'Lou Dobbs')
+
+        # Put Lou Dobbs in a skeezy hotel.
+        hotel_room = room.Room.create()
+        hotel_room.add_character(lou_dobbs)
+        assert(lou_dobbs in hotel_room.characters)
```

Comment:
Be consistent with assert usage. Other places in the file you use unittests' `assertIn` method, but not here. Pick one and stick to it.

Reply:
So these aren't the actual test assertions. I use bare `assert` for preconditions that aren't directly the responsibility of the code being tested. These two propositions will be addressed (and properly tested with `assert(Not)?In` when we add tests for `add_character`.

Reply:
Hmm, can these be removed then? I'd prefer to remove the preconditions from this test now, since it's likely that we won't remember to take them out later.

Reply:
I don't intend for them to be removed.

My goal with this kind of thing is to avoid subtle/incomprehensible failures in future. For example, if `add_character` stops working at some point, and these conditions are no longer true, then, when `die()` is called in this test, it will throw a `KeyError`. I'd rather see one of these assertions fail so that it's more clear where the problem is.

Some testing frameworks offer different constructs for this ("expect" vs. "assert"), but none of Python's testing frameworks do.

Reply:
Can I resolve this? Do you strongly disagree with the retention of these preconditions?